index.html

```diff
@@ -321,6 +323,7 @@
             <li>getAvailability()'s promise resolves with a RemotePlaybackAvailability,
               similar to the function with same name in Presentation API.</li>
             <li>state returns the remote playback connection's current state.</li>
+            <li>statechange notifies about the change of the above.</li>
```

Comment:
No need for that. We should actually drop the "draft" summary. It was meant because there was no real spec :)